espnet/nets/pytorch_backend/e2e_tts_transformer.py

```diff
@@ -325,7 +277,8 @@ def __init__(self, idim, odim, args=None):
                 - eprenet_dropout_rate (float): Dropout rate in encoder prenet.
                 - dprenet_dropout_rate (float): Dropout rate in decoder prenet.
                 - postnet_dropout_rate (float): Dropout rate in postnet.
-                - use_masking (bool): Whether to use masking in calculation of loss.
+                - use_masking (bool): Whether to apply masking for padded part in loss calculation.
+                - use_weighted_masking (bool): Whether to apply weighted masking for padded part in loss calculation.
```

Comment:
Same as the above; a bit confusing. Weighted mask (`w`) is applied for the non-padded part as well (0 is for the padded part) in my understanding.

Reply:
Right. I removed `for padded part`.